src/util/transfer_metadata.hpp

```diff
@@ -0,0 +1,17 @@
+/**
+ * This software is distributed under the terms of the MIT License.
+ * Copyright (c) 2020-2023 LXRobotics.
+ * Author: Alexander Entinger <alexander.entinger@lxrobotics.com>
+ * Contributors: https://github.com/107-systems/107-Arduino-Cyphal/graphs/contributors.
+ */
+
+#ifndef ARDUINO_CYPHAL_TRANSFER_METADATA_HPP_
+#define ARDUINO_CYPHAL_TRANSFER_METADATA_HPP_
+
+struct TransferMetadata final
+{
+  uint16_t node_id;
```

Comment:
Should probably be called `remote_node_id` to be consistent with the relevant libcanard metadata entry.